tests/rule_based_profiler/bobby_user_workflow_fixture.py

```diff
@@ -364,14 +371,14 @@ def bobby_columnar_table_multi_batch():
                         }
                     }
                 },
-            },
+            }
         ),
         ExpectationConfiguration(
             **{
                 "kwargs": {
                     "column": "fare_amount",
-                    "min_value": -1982.49,
-                    "max_value": 5201.49,
+                    "min_value": 0.0,
+                    "max_value": 6689.35,
```

Comment:
I'm kinda floored that looking at max by month is giving us a lower bound of 0. Thoughts?

Reply:
@jcampbell It is the same issue -- we have large swings in the data, resulting in a high standard deviation.  My previous comment suggests building a cleaner data set -- happy to discuss.  Thanks!